server/routes/where-is-item.route.js

```diff
@@ -1,42 +1,64 @@
 'use strict'
 
-const { Paths, Views } = require('../utils/constants')
-const { buildErrorSummary } = require('../utils/validation')
+const RedisService = require('../services/redis.service')
+const { Options, Paths, RedisKeys, Views } = require('../utils/constants')
+const { buildErrorSummary, Validators } = require('../utils/validation')
 
 const handlers = {
-  get: (request, h) => {
+  get: async (request, h) => {
     return h.view(Views.WHERE_IS_ITEM, {
-      ..._getContext()
+      ...(await _getContext(request))
     })
   },
 
-  post: (request, h) => {
+  post: async (request, h) => {
     const payload = request.payload
     const errors = _validateForm(payload)
 
     if (errors.length) {
       return h
         .view(Views.WHERE_IS_ITEM, {
-          ..._getContext(),
+          ...(await _getContext(request)),
```

Comment:
Don't think you need the await or to pass the request for _getContext.

Reply:
Fixed.